### ISSUE TITLE
bsdtar: ignore UTF-8 encoding errors

### ISSUE THREAD

**Issue by pohly** [clearlinux/swupd-client#280]
When trying with current refkit, swupd verify --install fails with:
```
	...100%zero pack downloads failed. 
Failed - Server-side error, cannot download necessary files
```

The actual error is not visible because stderr gets redirected to /dev/null (also for GNUtar - perhaps that should be changed?).

Manually unpacking the pack-os-core-from-0-to-20.tar with bsdtar reveals the actual root cause:
```
bsdtar: Linkname can't be converted from UTF-8 to current locale.
bsdtar: Linkname can't be converted from UTF-8 to current locale.
bsdtar: Linkname can't be converted from UTF-8 to current locale.
bsdtar: Linkname can't be converted from UTF-8 to current locale.
bsdtar: Error exit delayed from previous errors.
```

This is related to file name encoding, see https://github.com/libarchive/libarchive/wiki/Filenames

In my case, no locale is set and thus conversion from UTF-8 into 7-bit ASCII (or whatever the default is) fails.

I'm not sure yet what the proper solution is. Perhaps the system simply has to have a UTF-8 locale available, and it needs to be active.

**Comment by matthewrsj** [comment]
Looks like the client where we did the libarchive testing is running in UTF-8 locale:
```
# localectl
   System Locale: LANG=en_US.UTF-8
       VC Keymap: n/a
      X11 Layout: n/a
```

**Comment by matthewrsj** [comment]
This will be resolved by #327 once it is merged.

**Comment by matthewrsj** [comment]
#327 has been merged. there is a similar issue being tracked in clearlinux/mixer-tools#92
